setup.cfg

```diff
@@ -2,3 +2,11 @@
 ignore =
     *.cfg
     bootstrap.py
+
+[isort]
+force_alphabetical_sort = True
+force_single_line = True
+lines_after_imports = 2
+line_length = 200
+not_skip = __init__.py
+
```

Comment:
I would rather not keep that here, and instead, reference https://raw.githubusercontent.com/plone/plone.recipe.codeanalysis/master/.isort.cfg

If we ever change the sorting order, please don't :D, jenkins will be able to detect and report those changes which could potentially be different from what is here, and I don't want to go around all +300 packages looking which ones do have their own configuration and update them :-/

If, eventually, p.r.codeanalysis has all that information on itself and does not rely on external configuration files (or a centrally managed one) then it would be even better :-)